<?xml version="1.0"?>
<!--
Copyright (c) 2019, I3D Robotics Ltd.
-->

<robot xmlns:xacro="http://ros.org/wiki/xacro">
    <xacro:macro name="aruco_marker" params="name size:=1">
        <link name="${name}_aruco_marker">
            <visual>
                <origin xyz="0 0 0" rpy="0 0 0"/>
                <geometry>
                    <box size="${size} ${size} 0"/>
                </geometry>
                <material name="${name}_marker_colour">
                    <color rgba="1 0 0 1"/>
                </material>
            </visual>
        </link>
    </xacro:macro>
</robot>